Asterisk.2013/Asterisk.NET/Helper.cs

```diff
@@ -754,6 +754,10 @@ internal static ManagerResponse BuildResponse(Dictionary<string, string> attribu
             int hash, hashEvent;
 
             eventType = attributes["event"].ToLower(CultureInfo);
+            if (eventType == "queuememberpause") //temporary fix for Asterisk 13 pause QueueMemberPause

```

Comment:
Yeah, we can have a look at [`Obsolete`](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx)